tools/continuous-ingestor/TestRunbook.ipynb

```diff
@@ -0,0 +1,1207 @@
+    "display.display(result)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# 20 server metric values for 10 different machines/dimension sets   =>  200 unique metric time series\n",
+    "#  5 process event values from 12 different processes/dimension sets =>   60 unique event time series\n",
+    "\n",
+    "# AUTOSTOP after 5 iterations, 260 * 5 = 1300\n",
+    "query = \"\"\"SELECT count(1) as cnt\n",
+    "             FROM {}.{}\"\"\".format(DB_NAME, TABLE_NAME6)\n",
+    "result1 = timestream.executeQueryAndReturnAsDataframe(read_client, query, True)\n",
+    "#display.display(result1)\n",
+    "\n",
+    "query = \"\"\"SELECT count(distinct region) as cnt\n",
+    "             FROM {}.{}\"\"\".format(DB_NAME, TABLE_NAME6)\n",
+    "result2 = timestream.executeQueryAndReturnAsDataframe(read_client, query, True)\n",
+    "#display.display(result2)\n",
+    "\n",
+    "query = \"\"\"SELECT count(1) as cnt, region\n",
+    "             FROM {}.{}\n",
+    "            GROUP BY region\"\"\".format(DB_NAME, TABLE_NAME6)\n",
+    "result3 = timestream.executeQueryAndReturnAsDataframe(read_client, query, True)\n",
+    "display.display(result3)\n",
+    "\n",
+    "\n",
+    "if result1['cnt'][0] != 1300:\n",
+    "    print('\\x1B[91m' + 'TEST 6-1: FAILED: We expect 1300 records. Actual {}'.format(result1['cnt'][0]))\n",
+    "else:\n",
+    "    print('\\x1b[92m' + 'TEST 6-1: SUCCESS: We expect 1300 records.')\n",
+    "\n",
+    "    \n",
+    "if result2['cnt'][0] != 1:\n",
+    "    print('\\x1B[91m' + 'TEST 6-2: FAILED: We expect exactly one region. Actual {}'.format(result2['cnt'][0]))\n",
+    "else:\n",
+    "    print('\\x1b[92m' + 'TEST 6-2: SUCCESS: We expect exactly one region.')\n",
+    "\n",
+    "if result3['region'][0] != 'eu-central-1':\n",
+    "    print('\\x1B[91m' + 'TEST 6-3: FAILED: We expect <eu-central-1> as region. Actual {}'.format(result3['region'][0]))\n",
+    "else:\n",
+    "    print('\\x1b[92m' + 'TEST 6-3: SUCCESS: We expect <eu-central-1> as region.')\n",
+    "\n",
+    "if result3['cnt'][0] != 1300:\n",
+    "    print('\\x1B[91m' + 'TEST 6-4: FAILED: We expect 1300 records in <eu-central-1> region. Actual {}'.format(result3['cnt'][0]))\n",
+    "else:\n",
+    "    print('\\x1b[92m' + 'TEST 6-4: SUCCESS: We expect 1300 records in <eu-central-1> region.')\n",
+    "    \n",
+    "print('\\x1b[0m' + 'TEST 6: DONE') \n",
+    "    "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### TEST 7: Include Only Athena Microservice\n",
+    "(Back to [top](#Contents))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "###################################################################################################\n",
+    "## TEST 7: Run ingestor for data in 'eu-central-1' only (1 cell, 1 silo)\n",
```

Comment:
This description is identical to `TEST 6`'s. You can add `and 'athena' service` to the end to help identify